view/mxGraph.d.ts

```diff
@@ -1454,7 +1454,7 @@ declare class mxGraph extends mxEventSource {
    * @param cells Optional array of {@link mxCells} to be grouped. If null is specified
```

Comment:
This fix in the TSDoc could be added
```suggestion
   * @param cells Optional array of {@link mxCell} to be grouped. If null is specified
```